R/check-col-names.R

```diff
@@ -27,20 +27,22 @@ check_col_names <- function(data, template, success_msg = NULL, fail_msg = NULL,
 #' @export
 #' @rdname check_col_names
 #' @examples
-#'
+#' \dontrun{
```

Comment:
```suggestion
#' \dontrun{
#' library("synapser")
#' synLogin()
```

I'd suggest adding these two lines so that someone could directly run the example, otherwise looks great!